server/ntb/publish/ntb_nitf.py

```diff
@@ -148,6 +225,16 @@ def repl_embedded(match):
         # and etree from stdlib doesn't have a proper HTML parser
         soup = BeautifulSoup(html, 'html.parser')
         html_elts = ET.fromstring('<div>{}</div>'.format(soup.decode(formatter='xml')))
+        # <p class="lead" lede="true"> is used by NTB for abstract
```

Comment:
afaik it's part of nitf standard, maybe they want to have it there. (not the class, but the `lede=true`)